### PR TITLE
[nuvo] Fix MPS4 flag and online status reporting

### PR DESCRIPTION
**Pull request by mlobstein**
Fix two minor issues:

1. When thing is provisioned from the UI, the port config item was populated with a default value. This caused the MPS4 flag to be set erroneously even when using serial port connections.
2. For serial port connections, the thing was always showing as ONLINE even when the serial port was not actually connected to the amplifier.

### CODE REVIEW

File: bundles/org.openhab.binding.nuvo/src/main/java/org/openhab/binding/nuvo/internal/handler/NuvoHandler.java [hunk 1]
```diff
@@ -598,21 +597,15 @@ private void scheduleReconnectJob() {
...
                     }
                 } else {
-                    error = "Reconnection failed";
-                }
-                if (error != null) {
-                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, error);
+                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, "Reconnection failed");
                     closeConnection();
-                } else {
-                    updateStatus(ThingStatus.ONLINE, ThingStatusDetail.NONE, this.versionString);
```
Comment (lolodomo) [inline bundles/org.openhab.binding.nuvo/src/main/java/org/openhab/binding/nuvo/internal/handler/NuvoHandler.java:615]: You want now to set the thing status ONLINE only when something is received through the connection ?
In practice, will a message be received systematically quickly after the connection ?

Comment (mlobstein) [reply bundles/org.openhab.binding.nuvo/src/main/java/org/openhab/binding/nuvo/internal/handler/NuvoHandler.java:615]: Yes, for serial port connections, pollStatus() will cause messages to be received almost immediately. For MPS4, a ping message will arrive within a few seconds.

Comment (lolodomo) [reply bundles/org.openhab.binding.nuvo/src/main/java/org/openhab/binding/nuvo/internal/handler/NuvoHandler.java:615]: Ok

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**